README.md

```diff
@@ -66,6 +75,8 @@ cluster/kubectl.sh get nodes
 
 To setup a multi-node cluster, please refer to [Arktos Cluster Setup Guide](docs/setup-guide/multi-node-dev-cluster.md). And [this guide](docs/setup-guide/arktos-apiserver-partition.md) gives detailed instructions if you want to enable partitions in the cluster.
 
+To setup an arktos scale out cluster, please refer to [Setting up local dev environment for scale out](docs/setup-guide/scale-out-local-dev-setup.md).
```

Comment:
this is in readme we need to be careful on our recommended ways to setup the scale-out clusters.

I think we need to push to use the kube-up.sh version of the setup for scale-up and scale-out clusters. that was the tool we will push for officially.

the local dev toolset is useful and we can surely add it here. but we need to be clear for their purposes.

Reply:
I don't think we have an official document for kube-up.sh manual.

Reply:
Added instruction for kube-up